packages/workspace/src/core/project-graph/project-graph.ts

```diff
@@ -38,30 +39,85 @@ import {
   buildWorkspaceProjectNodes,
 } from './build-nodes';
 
+export const NEXT_GRAPH_VERSION = '4.0';
```

Comment:
Why do you need these constants?

Cause they do create a level of indirection which is somewhat problematic. If a piece of code depends on the version 3 of the graph, then updating Nx will change 3 to 4 and the piece of code will not work anymore.

I think it's better to explicitly pass '3.0' cause then no indirection, the scenario above is not a problem.

Reply:
My reasoning was to avoid typos and help with easier search & replace later. But your point makes sense.